modules/core/src/shaderlib/picking/picking.ts

```diff
@@ -36,5 +32,15 @@ uniform bool picking_uAttribute;
     `
     }
   },
-  ...picking
-} as ShaderModule<PickingModuleSettings>;
+  ...picking,
+  // TODO migrate to luma option names
```

Comment:
> // TODO migrate to luma option names

I could agree with this, it could make sense to remove PickingModuleSettings as it appears to be an internal class. 

We have two well-defined APIs that we care about:
- layer props
- luma shader module props

We want a type safe mapping between these that is reasonably centralized, but having an intermediary set of types  in `PickingModuleSettings` does not seem to help readability at least based on what I see in this PR.